### PR TITLE
Milestone 1

### PR DESCRIPTION
**Pull request by ZJUAlexChen**
This is for the first Pull Request with readme file finished.

### CODE REVIEW

File: README.md
Comment (ashuk203) [inline README.md:4]: For you functional design, please use `snake_case` for your python functions (you are currently using `camelCase`)

File: README.md
Comment (ashuk203) [inline README.md]: Could you provide add more detail / clarification of what the `data` argument? Is it the training data?

**Review by ashuk203 — CHANGES_REQUESTED** [review]
Apart from the few changes requested in the comments, everything looks good!

File: README.md
Comment (ZJUAlexChen) [reply README.md:4]: I have changed the camelCase to snake_case. Thanks.

File: README.md
Comment (ZJUAlexChen) [reply README.md]: I have clarified the data argument as the training data. Thanks. I wonder if I should merge the request by myself after your second review, or you merge it after reviewing my correction?

**Review by ashuk203 — APPROVED** [review]

**Comment by ashuk203** [comment]
You can go ahead and merge your pull request. Good work!

**Pull request merged**
